### PR TITLE
Split service.events into forge-specific sub-modules

### LINKED ISSUE #672: Split `service.events` into forge-specific sub-modules
Classes in `service.events` have a repetitive naming (`*GitlabEvent`, `*GithubEvent`). This might be a good indication to split the module into multiple, forge-specific sub-modules, for example: `service.events.github` and `service.events.gitlab`.

This was discussed many times (for example #671).

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #672

### CODE REVIEW

File: packit_service/service/events/event.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+"""
+This file defines classes for events which are sent by GitHub or FedMsg.
+"""
```
Comment (lachmanfrantisek) [inline packit_service/service/events/event.py:6]: Is this still relevant?

**Review by lachmanfrantisek — APPROVED** [review]
You are hero! :rocket:

File: packit_service/service/events/__init__.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    MergeRequestGitlabEvent.__name__,
+    MergeRequestCommentGitlabEvent.__name__,
+    IssueCommentGitlabEvent.__name__,
+    KojiBuildEvent.__name__,
+    AbstractPagureEvent.__name__,
+    PushPagureEvent.__name__,
+    PullRequestCommentPagureEvent.__name__,
+    PullRequestPagureEvent.__name__,
+    TestingFarmResultsEvent.__name__,
+]
```
Comment (TomasTomecek) [inline packit_service/service/events/__init__.py:56]: +1 for keeping the imports working

**Review by TomasTomecek — APPROVED** [review]
Very nice!

I'd also like to point out that modules `github`, `gitlab`, `koji` exist so we should be extra cautious with imports (which are handled by our IDEs, right? and verified by mypy, so probably not a problem after all)

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
recheck

**Comment by morucci** [comment]
recheck

**Comment by jpopelka** [comment]
recheck

**Pull request merged**